src/effects/effectsmanager.cpp

```diff
@@ -177,6 +177,8 @@ void EffectsManager::readEffectsXml() {
     }
     file.close();
 
+    // QuickEffect chains are created only for existing main decks, so read
+    // the configured presets only for those
```

Comment:
```suggestion
    // Note: QuickEffect chains are created only for existing main decks
```

The second half sound as if we need to special case something here, but that is not the case. The comment only explains the surprising file name, right?

Reply:
> The comment only explains the surprising file name, right?

Hm?

I think the imperative is misleading here.
```suggestion
    // Note: QuickEffect chains are created only for existing main decks
    // thus only for those the configured presets are requested
```
Better?

Reply:
Yes, Thank you.